docs/development/extensions-core/hdfs.md

```diff
@@ -94,7 +94,7 @@ For more configurations, see the [Hadoop AWS module](https://hadoop.apache.org/d
 
 #### Configuration for Google Cloud Storage
 
-To use the Google cloud Storage as the deep storage, you need to configure `druid.storage.storageDirectory` properly.
+To use the Google Cloud Storage as the deep storage, you need to configure `druid.storage.storageDirectory` properly.
```

Comment:
For the installation section below, I think we could point to https://github.com/GoogleCloudPlatform/bigdata-interop/blob/master/gcs/INSTALL.md and say the following, and remove the parts where we duplicate their setup instructions:

> Please follow the instructions at https://github.com/GoogleCloudPlatform/bigdata-interop/blob/master/gcs/INSTALL.md for configuring your `core-site.xml` with the filesystem and authentication properties needed for GCS."

We can also add the following (it took me a while to find a download link for the connector): 

> The GCS connector library is available at https://cloud.google.com/dataproc/docs/concepts/connectors/cloud-storage#other_sparkhadoop_clusters


The line below:
"Tested with Druid 0.9.0, Hadoop 2.7.2 and gcs-connector jar 1.4.4-hadoop2."

can be updated to

"Tested with Druid 0.17.0, Hadoop 2.8.5 and gcs-connector jar 2.0.0-hadoop2.

Reply:
Thanks, I added suggestions. But I would still want to keep the example properties for GCS, since they are pretty mandatory. The similar pattern is applied to [S3 configuration](https://github.com/apache/druid/pull/9171/files#diff-51abd0f049462a98772db4c6ea063be3R66-R93).